src/main/java/frc/team2412/robot/sensors/AprilTagsProcessor.java

```diff
@@ -72,39 +63,16 @@ private static boolean hasValidRotation(Transform3d camToTarget) {
 				&& (Math.abs(robotToTargetRotation.getY()) < ROBOT_TO_TARGET_PITCH_TOLERANCE);
 	}
 
-	private static PhotonTrackedTarget swapBestAndAltTransforms(PhotonTrackedTarget target) {
-		return new PhotonTrackedTarget(
-				target.getYaw(),
-				target.getPitch(),
-				target.getArea(),
-				target.getSkew(),
-				target.getFiducialId(),
-				target.getAlternateCameraToTarget(), // Swap
-				target.getBestCameraToTarget(),
-				target.getPoseAmbiguity(),
-				target.getMinAreaRectCorners(),
-				target.getDetectedCorners());
-	}
-
-	private static PhotonPipelineResult filteredPipelineResult(PhotonPipelineResult result) {
-		var copy = copy(result);
-		for (int i = copy.targets.size() - 1; i >= 0; --i) {
-			var target = copy.targets.get(i);
+	private static boolean resultIsValid(PhotonPipelineResult result) {
+		for (var target : result.targets) {
```

Comment:
@KangarooKoala I realized that we could probably use Java's [Stream API](https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html) for this, but that's super unnecessary for competition (this is a random 1 am thought, just advertising as "this is something cool we could do later in the future post Sammamish").

Reply:
I've heard the stream API is pretty inefficient (though this one wouldn't need that many intermediate objects so maybe it wouldn't be as bad), but yeah we could look into that later.